pkg/certificate/providers/config.go

```diff
@@ -76,6 +79,8 @@ func NewCertificateProviderConfig(kubeClient kubernetes.Interface, kubeConfig *r
 		tresorOptions:      tresorOptions,
 		vaultOptions:       vaultOptions,
 		certManagerOptions: certManagerOptions,
+
```

Comment:
nit: remove extra line

Reply:
Will address this in a follow-up with additional cleanups.